german/schwimmen/schwimmen.md

```diff
@@ -5,13 +5,13 @@ Sie erreichen Sie mit letzter Kraft und erholen sich erst einmal am Strand. Doch
 Man sollte ...
 
 ... was zu Essen suchen, denn nach dem Schwimmen hat man immer Hunger
-[Hunger](hunger/hunger.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.